### DISCUSSION TITLE
enable devcontainer up in detached mode

### DISCUSSION THREAD

**Discussion by Jogi7819** [devcontainers/community#31] [Ideas]
devcontainer up runs per default the command docker run with `-a STDOUT` and -a `STDERR`. The command is executed without issues but the bash gets never released and stucks. It would by nice to allow instead of the mentioned paramters to run devcontainer up in detach mode which should result in `docker run --detach ... `

**Comment by Chuxel** [comment]
I created https://github.com/devcontainers/cli/issues/430 in the CLI repo to capture the feature request!
